tools/reduce/reduce.cpp

```diff
@@ -169,6 +178,15 @@ ReduceStatus ParseFlags(int argc, const char** argv,
             static_cast<uint32_t>(strtol(split_flag.second.c_str(), &end, 10));
         assert(end != split_flag.second.c_str() && errno == 0);
         reducer_options->set_step_limit(step_limit);
+      } else if (0 == strncmp(cur_arg, "--target-function=",
+                              sizeof("--target-function=") - 1)) {
+        const auto split_flag = spvtools::utils::SplitFlagArgs(cur_arg);
+        char* end = nullptr;
+        errno = 0;
+        const auto target_function =
+            static_cast<uint32_t>(strtol(split_flag.second.c_str(), &end, 10));
+        assert(end != split_flag.second.c_str() && errno == 0);
+        reducer_options->set_target_function(target_function);
```

Comment:
Could alternatively do something like this (for input `temp`). This is from gf-layers.

```
  std::istringstream ss{temp};
  std::uint64_t temp_uint{};
  ss >> temp_uint;

  if (ss.fail()) {
    LOG("Failed to parse %s", temp);
    return false;
  }

  *value = temp_uint;
  return true;
```

Reply:
This looks good, but I think we should do it in a separate PR and for all the options.